src/common/telemetry_event/src/lib.rs

```diff
@@ -95,15 +117,8 @@ pub fn request_to_telemetry_event(
         node,
         is_test,
     };
-    let report_bytes = event.encode_to_vec();
-
-    tokio::spawn(async move {
-        const TELEMETRY_EVENT_REPORT_TYPE: &str = "event";
-        let url = (TELEMETRY_REPORT_URL.to_owned() + "/" + TELEMETRY_EVENT_REPORT_TYPE).to_owned();
-        post_telemetry_report_pb(&url, report_bytes)
-            .await
-            .unwrap_or_else(|e| tracing::info!("{}", e))
-    });
+
+    TELEMETRY_EVENT_REPORT_STASH.blocking_lock().push(event);
```

Comment:
Given that we call `blocking_lock` here, why not just using a non-async mutex (`parking_lot::Mutex`) instead of the tokio async mutex?

Reply:
after switching to unbounded channel, there seems no need for the sync Mutex.